### DISCUSSION TITLE
Can we make animated app icon

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#162] [Developer Feedback]
I was just wondering to make it.

**Comment by XiaomaiTX** [comment]
Likewise, I'm looking forward to the system using this ability

**Comment by liqin-ZeppHealth** [comment]
hi @alexmathew11 , @XiaomaiTX , thanks for your suggestion. by now we don't have any plan to support this feature. please know that, thank you.

**Comment by htoooth** [comment]
Currently unable to do it.
